webapp/www/greentrip_site/related_work.html

```diff
@@ -0,0 +1,22 @@
+<html>
+	<head>
+		<div class = "top_bar">
+			<div class = "item" id = "logo"> <a href = "index.html"> <img src = "assets/img/greentrip_logo.png" style = "width:150px; height:100px;"> </a> </div>
+			<div class = "item"> <a href = "incentive.html"> Incentivization </a> </div>
+			<div class = "item"> <a href = "who_we_are.html"> Who We Are </a> </div>
+			<div class = "item"> <a href = "supporters.html"> Supporters </a> </div>
+			<div class = "item"> <a href = "related_work.html"> Related Works </a> </div>
+			<div class = "button"> JOIN STUDY </div>
+		</div>
+	<link rel="stylesheet" type="text/css" href="assets/css/index.css">
+	<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=EB+Garamond" />
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+	</head>
+	<div class = "big_box" id = "related_work">
+		<div class = "large_text" id = "large_text_1"> Related Work </div> 
+		<div class = "smaller_text"> The previous group working with the emission study used polar bears as a representation of the emotional connectivity to reducing our environmental impact and used a simple suggestion system as a representation of informative connectivity. The suggestion system used for Greentrip was largely based on the previous group's suggestion system model.
```

Comment:
@sam11b aren't  you going to add a link to their tech report?
I thought I had a link on the google doc, but here it is again.
https://www2.eecs.berkeley.edu/Pubs/TechRpts/2019/EECS-2019-2.html